### DISCUSSION TITLE
TabHeader with Button

### DISCUSSION THREAD

**Discussion by Phunguy65** [AvaloniaUI/Avalonia#16500] [Q&A]
Hi, I try to put a Button into TabHeader property of TabItem.
But when I click TabHeader then SelectedPipe not  show under Button.
I also set Focusable=False, but it's not work.
How can I do something this issue?
Thanks everyone.

**Comment by maxkatz6** [comment]
It's not really clear what you are trying to make.
If you want to put a button in there, but disable its input - IsHitTestVisible=false together with Focusable=false should work.
